cassandra/cluster.py

```diff
@@ -3964,6 +4000,42 @@ def _refresh_node_list_and_token_map(self, connection, preloaded_results=None,
             log.debug("[control connection] Rebuilding token map due to topology changes")
             self._cluster.metadata.rebuild_token_map(partitioner, token_map)
 
+    def refresh_tablets(self, connection):
```

Comment:
I think better name would be something like `_refresh_tablets_periodically`.

Reply:
renamed